test/boostTest.cpp

```diff
@@ -121,15 +121,27 @@ int registerTests(
 	}
 	return numTestsAdded;
 }
+
+void initializeOptions()
+{
+	auto const& suite = boost::unit_test::framework::master_test_suite();
+
+	auto options = std::make_unique<solidity::test::CommonOptions>();
+	solAssert(options->parse(suite.argc, suite.argv), "Failed to parse options!");
+	options->validate();
+
+	solidity::test::CommonOptions::setSingleton(std::move(options));
+}
 }
 
 test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 {
 	master_test_suite_t& master = framework::master_test_suite();
 	master.p_name.value = "SolidityTests";
-	solidity::test::Options::get().validate();
 
-	bool disableSemantics = !solidity::test::EVMHost::getVM(solidity::test::Options::get().evmonePath.string());
+	initializeOptions();
+
+	bool disableSemantics = !solidity::test::EVMHost::getVM(solidity::test::CommonOptions::get().evmonePath.string());
```

Comment:
Probably nit-picking: but what about `skipSemanticTests` ?

Reply:
Not sure.. I mean this is just renaming `Options` -> `CommonOptions`..
I can rename that in an extra commit if you want, sure.